scripts/eurostat/health_determinants/tobacco_consumption/README.md

```diff
@@ -0,0 +1,82 @@
+
+## About the Dataset
+This dataset has Population Estimates for the National geographic levels in Europe for the years 2008,2014,2019.
+
+The population is categorized by various set of combinations as below:
+        
+        1. Smoking of Tobacco Products by Sex and Educational Attainment level
+        2. Smoking of Tobacco Products by Sex and Income Quintile
+        3. Smoking of Tobacco Products by Sex and Degree Of Urbanisation
+        4. Daily Smokers of Cigarettes by Sex and Educational Attainment level
+        5. Daily Smokers of Cigarettes by Sex and Income Quintile
+        6. Daily Smokers of Cigarettes by Sex and Degree Of Urbanisation
+        7. Daily Exposure to Tobacco smoke indoors by Sex and Educational Attainment level
+        8. Daily Exposure to Tobacco smoke indoors by Sex and Degree Of Urbanisation
+        9. Smoking of Tobacco Products by Sex and Country of Birth
+        10. Smoking of Tobacco Products by Sex and Country of Citizenship
+        11. Former Daily Tobacco Smokers by Sex and Income Quintile
+        12. Former Daily Tobacco Smokers by Sex and Educational Attainment level
+        13. Duration of Daily Tobacco smoking by Sex and Educational Attainment level
+        14. Use of Electronic Cigarettes or similar electronic devices by Sex and Educational Attainment level 
+        15. Daily Smokers of Cigarettes by Sex and Educational Attainment level
+	16. Daily Smokers of Cigarettes by Sex and Income Quintile
+	17. Daily Smokers by number of Cigarettes by Sex and Educational Attainment level
+        
+
+### Download URL
+The data in tsv.gz formats are downloadable from https://ec.europa.eu/eurostat/web/health/data/database -> 	Health -> Health determinants (hlth_det).
+The actual URLs are listed in input_files.py.
+
+
+#### API Output
+These are the attributes that will be used
+| Attribute      		                        | Description                                                   |
+|-------------------------------------------------------|---------------------------------------------------------------|
+| time       					| The Year of the population estimates provided. 	                |
+| geo       					| The Area of the population estimates provided. 			|
+| Tobacco Consumption  				| The type of Tobacco Consumption. 					|
+| Educational Attainment level   	| The level of education of the population.  |
+| Sex   				| Gender either Male or Female. 						|
+| Income Quintile 				| The slab of income of the population.					|
+| Degree of Urbanisation   			| The type of residence (rural/urban/semiurban) of the population.      |
+| Country of Birth   				| The nativity of the population.					|
+| Country of Citizenship   				| The citizenship of the population.				|
+| Degree of Activity Limitation   				|  							|
+
+
+
+
+#### Cleaned Data
+Cleaned data will be inside [output/eurostat_population_tobaccoconsumption.csv] as a CSV file with the following columns.
+
+- time
```

Comment:
Same comment about time and geo and capital letters as above.

Reply:
Modified the names.